helpers/traits/AltapayMaster.php

```diff
@@ -146,7 +148,7 @@ public function scheduledSubscriptionsPayment( $amount, $renewal_order ) {
 	 */
 	public function apiLogin() {
 		if ( $this->api === false ) {
-			$this->api = new AltapayMerchantAPI(
+			$this->api = new \AltapayMerchantAPI(
```

Comment:
I would suggest declaring there use in the top of the file instead of prefixing root classes with \ inline